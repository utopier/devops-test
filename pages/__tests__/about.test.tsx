/* eslint-disable */
import React from 'react';
import About from '../about';
import { render, screen } from '@testing-library/react';

describe('about page', () => {
	it('matches snapshot', () => {
		const utils = render(<About />);
		expect(utils.container).toMatchSnapshot();
	});
	it('about text', () => {
		const utils = render(<About />);
		utils.findByText('about');
	});
});
