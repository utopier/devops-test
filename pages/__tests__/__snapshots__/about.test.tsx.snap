// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about page matches snapshot 1`] = `
<div>
  <h1>
    About
  </h1>
</div>
`;
